README.md

```diff
@@ -3,6 +3,11 @@ angular-gist
 
 AngularJS directive for embedding Github gists.
 
+## Demo
+Please have a look at the [demo](http://htmlpreview.github.io/?https://raw.githubusercontent.com/AWolf81/angular-gist/demo-app/examples/simple-demo/index.html) for a live example.
```

Comment:
@AWolf81 looks like we can just use a link this instead: https://github.com/scottcorgan/angular-gist/blob/master/examples/simple-demo/index.html